_styles/docs.css

```diff
@@ -200,7 +200,9 @@ nav {
     color: #403757;
 }
 
-.sidebar > ul {
+.sidebar > ul,
```

Comment:
Can we make this one like `.sidebar ul` to look like the others, or does it break css specificity?